google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionStatementExecutorImpl.java

```diff
@@ -83,6 +89,11 @@
  * calls are then forwarded into a {@link Connection}.
  */
 class ConnectionStatementExecutorImpl implements ConnectionStatementExecutor {
+
+  private final Set<String> explainOptions =
```

Comment:
nit:
```suggestion
  private static final Set<String> EXPLAIN_OPTIONS =
```

Also, we should consider moving this set to `ClientSideStatementExplainExecutor.java`, as it is specific for `explain`, and not something that is generically needed for all client side statements.